Description: The IPP backend could get stuck in an endless loop on certain network errors (STR #4194)
Date: Mon, 1 Oct 2012 02:10:26 +0000
Origin: http://svn.cups.org/public/cups/trunk@10623
Last-Update: 2012-11-06

--- a/backend/ipp.c
+++ b/backend/ipp.c
@@ -922,6 +922,8 @@
 	_cupsLangPrintFilter(stderr, "ERROR",
 	                     _("Unable to get printer status."));
         sleep(10);
+
+	httpReconnect(http);
       }
 
       ippDelete(supported);
